src/plugins/workspace/public/plugin.ts

```diff
@@ -275,32 +274,17 @@ export class WorkspacePlugin implements Plugin<{}, {}, WorkspacePluginSetupDeps>
     });
 
     /**
-     * register workspace overview page
+     * register workspace detail page
      */
     core.application.register({
-      id: WORKSPACE_OVERVIEW_APP_ID,
+      id: WORKSPACE_DETAIL_APP_ID,
       title: i18n.translate('workspace.settings.workspaceOverview', {
```

Comment:
Better to change the translate key as well.

Reply:
Sure, it has been updated